src/main/java/com/codeborne/selenide/SelenideConfig.java

```diff
@@ -345,6 +345,14 @@ public SelenideConfig browserCapabilities(MutableCapabilities browserCapabilitie
   }
 
   private String getProperty(String key, String defaultValue) {
-    return properties.getProperty(key, defaultValue);
+    String value = properties.getProperty(key, defaultValue);
+    return replaceEmptyStringWithNull(value, defaultValue);
+  }
+
+  private String replaceEmptyStringWithNull(String value, String defaultValue) {
```

Comment:
I find this method hard to understand.  :(

It's strange that we do ignore `defaultValue` in most cases, and use it only if it's `null`. 

Probably it would be easier to inline method `replaceEmptyStringWithNull`?

Reply:
Agree it is not obvious. Inlined.